cumulus/steps/ec2/ingress_rule.py

```diff
@@ -9,19 +9,17 @@ class IngressRule(step.Step):
 
     def __init__(self,
                  port_to_open,
-                 name,
                  cidr):
 
-        step.Step.__init__(self)
+        step.Step.__init__(self, name='Port%sIngressRule')
 
         self.port_to_open = port_to_open
         self.cidr = cidr
-        self.name = name
 
     def handle(self, chain_context):
         template = chain_context.template
         template.add_resource(ec2.SecurityGroupIngress(
-            self.name,
+            self.name % self.port_to_open,
```

Comment:
why is self.name being set here and not in the constructor?   flake8 should be catching this.. doesn't it?

Reply:
right self.name comes in from the base class.   It's kind of nasty to have some names go in with a replacement token that the class has to handle later on.   If I want to just use self.name later on I have to think about replacing it.

__init__()
```
name = 'Port%sIngressRule'  % self.port_to_open
 step.Step.__init__(self, name=name)

```